addons/spreadsheet_account/static/src/account_group_auto_complete.js

```diff
@@ -0,0 +1,51 @@
+import { _t } from "@web/core/l10n/translation";
+
+import { registries, tokenColors, helpers } from "@odoo/o-spreadsheet";
+
+const { insertTokenAfterLeftParenthesis } = helpers;
+
+// copy-pasted list of options from the `account_type` selection field.
+const ACCOUNT_TYPES = [
```

Comment:
It should probably be fetched from the server to respect the extensibility but I'm not sure it's worth the trouble?

Reply:
Ideally yes, but the auto-complete mechanism isn't designed to work with async right now.

I asked OLMA if that field was extended in some localisation and he said no. And looking at the code, I come to the same conclusion

(we could also say it's a better UX to have the auto-complete instantly rather than having to wait for a network round-trip)